### PR TITLE
Improve process selection UX

### PR DESCRIPTION
**Pull request by Aurumaker72**
Helps recognizability when multiple instances are open and in different states

### CODE REVIEW

**Comment by FramePerfection** [comment]
When would this alone currently actually be beneficial? As far as I'm aware, the main window title is always just "STROOP", nothing like "STROOP (sm64.us.z64)" or the like. Would you mind explaining / implementing the idea you have for this?

**Comment by Aurumaker72** [comment]
This is about mupen window titles, not stroop. Mupen's titles can vary so having that degree of separation can help distinguishing versions and states which otherwise would have been all shown as the same process name.

Consider the following difference:

Old:
```
1. mupen64
2. mupen64
3. mupen64
```

New:
```
1. Mupen 64 1.1.8 - SUPER MARIO 64 - movie.m64
2. Mupen 64 1.0.10
3. Mupen 64 1.1.3 - AMA
```

**Comment by FramePerfection** [comment]
Ok wow, I didn't even realize this was at all what this was about. Sorry for the slow response btw, I've only just gotten around to check this out, and wouldn't the reality about this be a cruel joke, after all this time spent on not thinking in the right direction from my side:

grafik

This is with internal ROM names (I think that's what's being displayed?) `SUPER MARIO 64` and `SUPER MARIO 74EE`.  
The box doesn't scale or scroll, so there is actually no way to see the difference after all.

Would you care to addess this issue, please?

On a related note, once connected, the status indicator that says so should probably display the window title also (this is after connecting to... either of the two, see, I don't even know 💯):
grafik

**Comment by Aurumaker72** [comment]
> The box doesn't scale or scroll, so there is actually no way to see the difference after all.

This issue has been addressed by widening the listbox as well as the buttons below (24b140b290ecf6644a4af942d13d14e071580a04)

> the status indicator that says so should probably display the window title also 

c52aa27f718c92d816d54290f98596b9cf34d3f4

File: STROOP/STROOP.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <RootNamespace>STROOP</RootNamespace>
     <AssemblyName>STROOP</AssemblyName>
     <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <LangVersion>11</LangVersion>
```
Comment (FramePerfection) [inline STROOP/STROOP.csproj:13]: This breaks support for Visual Studio 2017. While I understand the want to use some of the more concise syntax that newer C# versions bring, that change should be part of a separate issue that would also specify and apply related coding styles (e.g. nullability annotations for reference types, pattern matching vs. equality checks, etc.).

**Review by FramePerfection — CHANGES_REQUESTED** [review]

**Review by FramePerfection — APPROVED** [review]

**Pull request merged**